backend/src/main/java/com/huellapositiva/infrastructure/orm/model/EmailConfirmation.java

```diff
@@ -27,10 +29,15 @@
     @Column(name = "hash", unique = true)
     private String hash;
 
-    @CreationTimestamp
+    @UpdateTimestamp
     @Column(name = "created_on")
-    private Date createdOn;
+    private Instant createdOn;
```

Comment:
Leave this column as it was. The idea is to have another column called updatedOn. CreatedOn date is only set when the entity is created in the database and should never ever be overriden again. To track when an entity is changed you have the column Date updatedOn. Do not forget to create the incremental flyway script that add the new column to the table